deps/uv/unofficial.gni

```diff
@@ -87,11 +87,11 @@ template("uv_gn_build") {
       ]
     }
     if (is_posix) {
-      libs = [ "m" ]
       ldflags = [ "-pthread" ]
     }
     if (is_linux) {
       libs += [
+        "m",
```

Comment:
This is not aligned with the gyp version, should this be behind `is_posix && !is_mac`

Reply:
Aligning would be better but as far as I understand the GN build only supports Windows, macOS and Linux anyways, so not exactly necessary.